.github/workflows/publish.yml

```diff
@@ -27,13 +29,13 @@ jobs:
           make -f Makefile.eve BRANCH?=${GITHUB_REF##*/} kernel-gcc
 
       - name: Log in to Docker Hub
-        if: github.ref == 'refs/heads/eve-kernel-amd64-v6.1.38-generic'
+        if: ${{ github.ref == env.branch_name }}
```

Comment:
can we use the env.branch_name here ?
`if: ${{ github.event.review.state == 'approved' || github.ref == 'refs/heads/eve-kernel-amd64-v6.1.38-generic' }}`

Reply:
I replaced it but then the GHA is complaining that the variable replacement is invalid. Figuring out a different way to fix this.

Reply:
@rucoder fixed.